examples/scalajs-react/src/main/scala/todomvc/types.scala

```diff
@@ -0,0 +1,29 @@
+package todomvc
+
+import java.util.UUID
+
+case class TodoId(id: UUID)
+
+object TodoId {
+  def random = new TodoId(UUID.randomUUID)
+}
+
+case class Title(value: String) {
+  def editable = UnfinishedTitle(value)
+}
+
+case class UnfinishedTitle(value: String){
```

Comment:
Nit, but could we get a space after the closing bracket? There are a few other places in the diff, too.